packages/transformer/src/IModelTransformer.ts

```diff
@@ -1684,6 +1691,7 @@ export class IModelTransformer extends IModelExportHandler {
     await this.initialize(options);
     await this.exporter.exportChanges(options);
     await this.processDeferredElements(); // eslint-disable-line deprecation/deprecation
+    await (this.exporter as any).exportAllAspects();
```

Comment:
typescript allows you to type-safely access private members using bracket syntax:
```ts
await this.exporter["exportAllAspects"();
```
Prefer this since it is type checked. You may need to tell the linter to shut up about the property access syntax, not sure if I merged the test branch where I configured the linter to allow it in the private/protected property case.

Reply:
Thanks, forgot that this can be used to access properties.